packages/compass-components/src/components/resizeable-sidebar.tsx

```diff
@@ -11,18 +11,56 @@ const containerStyles = css({
   maxWidth: '80%',
   height: '100%',
   position: 'relative',
-  background: uiColors.gray.dark3,
-  color: 'white',
+});
+
+const containerStylesDark = css({
```

Comment:
Once we just have darkmode support everywhere and both sidebars are only dark when darkmode is true we should probably move these colours even higher up so the whole app can use it?

Reply:
I think there are some trade offs using css vars, like ensuring you have the right name for a variable, and ensuring the vars you're creating are actually used. We might want to investigate how we can better do those things before using them all over just yet. Looks useful for this implementation though.